src/main/java/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,33 @@
+package impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+import model.Operation;
+import service.DataConverter;
+
+public class DataConverterImpl implements DataConverter {
+    public static final String SEPARATOR = ",";
+    public static final int HEADER_INDEX = 0;
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int FRUIT_TYPE_INDEX = 1;
+    private static final int FRUITS_QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> parseData(List<String> dataFromFile) {
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+        dataFromFile.remove(HEADER_INDEX);
```

Comment:
Directly modifying the input list by removing an item can lead to unexpected behavior and bugs, especially if this list is used elsewhere in the program. It's better to skip the first line during the iteration instead.